dynamic_programming/fast_fibonacci.py

```diff
@@ -8,14 +8,14 @@
 
 
 # returns F(n)
-def fibonacci(n: int):  # noqa: E999 This syntax is Python 3 only
+def fibonacci(n: int) -> int:  # noqa: E999 This syntax is Python 3 only
```

Comment:
Please lose the `noqa`  comments here and below because Python 2 is dead.

Reply:
As encouraged in the **Coding Style** section of [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md)
> Please write in Python 3.7+

So I agree with @cclauss comment.